Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:12
```

Comment:
This file should not be in the root folder as it is only used for generating this library. Maybe a folder like `gen`?

Reply:
Done.